src/test/java/org/tikv/BaseRawKVTest.java

```diff
@@ -21,6 +21,21 @@
 import org.tikv.util.TestUtils;
 
 public class BaseRawKVTest {
+
+  protected String getTiKVVersion() {
+    String tikvVersion = TestUtils.getEnv("TIKV_VERSION");
```

Comment:
Who will set the `TIKV_VERSION`?

Reply:
we can get the tikv version from TiKV, see https://github.com/tikv/client-java/blob/master/src/main/java/org/tikv/common/StoreVersion.java.

Reply:
Cool

Reply:
> Who will set the `TIKV_VERSION`?

By design, like RAWKV_PD_ADDRESSES, it will be set by the test excecutor.